### DISCUSSION TITLE
Now when git is removed...

### DISCUSSION THREAD

**Discussion by Dan1jel** [topgrade-rs/topgrade#663] [Q&A] [answered]
How do I add them as custom commands? Or how do I add all my (4) git folders. 

I do notice that one folder is pulled but not all folders. I tried to search for a result in my config file or on GitHub but didn't find any...?

**Comment by SteveLauC** [comment]
What is your configuration file? Have you set them [in this entry](https://github.com/topgrade-rs/topgrade/blob/ea13c51b7d74299a7a073a38a3d5f74f88f4fd65/config.example.toml#L159-L163)

**Reply by Dan1jel** [reply, marked as answer]
Ah sorry I thought git was removed completely, but I realised it's just some of the git settings that was removed. When I changed my folders from "pull_only_repos" to "repos" it worked again. Thanks for the quick reply, and yes next time I will create an issue, just thought it was more of a question then to start a "dumb" issue instead :)

**Comment by SteveLauC** [comment]
And, I recommend opening an issue, and provide:

1. The version of the Topgrade that you are using
2. The steps that you have tried
3. The result you got
4. The expected result
